packages/ember-application/lib/system/application.js

```diff
@@ -1003,6 +1004,7 @@ Application.reopenClass({
 
     registry.injection('view', 'renderer', 'renderer:-dom');
     registry.register('view:select', SelectView);
+    registry.register('view:outlet', OutletView);
```

Comment:
I think this should be prefixed with a dash, it is private and we don't want any apps having collisions with their `outlet` routes...